en/frames_multicopter/dji_flamewheel_450.md

```diff
@@ -0,0 +1,126 @@
+to set the board rotation correctly in *QGroundControl* afterwards.
+
+
+## Wiring and Connections
+
+General pinout for the Pixhawk 3 Pro can be found [here](https://pixhawk.drotek.com/en/inputs-outputs.html).
+
+### 3DR Telemetry
+
+The 3DR Telemetry does not come with a JST GH connector which the *Pixhawk 3 Pro* uses. The pinout remains the same and just the plug needs to be changed. Use the Telem 1 port on the *Pixhawk 3 Pro*.
+
+| pin | Pixhawk 3 Pro Telem 1 | 3DR Telemetry    |
+| --- | --------------------- | ---------------- |
+| 1   | VCC                   | VCC              |
+| 2   | TX                    | RX               |
+| 3   | RX                    | TX               |
+| 4   | CTS                   | CTS              |
+| 5   | RTS                   | RTS              |
+| 6   | GND                   | GND              |
+
+### Lidar-Lite V3
+
+The pinout for the *Lidar Lite V3* and the *Pixhawk 3 Pro* I2C 1 port is as follows.
+
+| pin | Pixhawk 3 Pro I2C 1 | Lidar Lite V3    |
+| --- | ------------------- | ---------------- |
+| 1   | VCC                 | VCC              |
+| 2   | SCL                 | - (Power enable) |
+| 3   | SDA                 | - (Mode control) |
+| 4   | GND                 | SCL              |
+| 5   | -                   | SDA              |
+| 6   | -                   | GND              |
+
+
+### Here+ RTK GPS
+
+The Here+ GPS comes with an 8 pin connector suited for the Pixhawk 2. In order to
+use it with the *Pixhawk 3 Pro* (or a Pixracer), a 6 pin connector has to be used
+and pin 6 and 7 are not needed (see below). The additional pins are for the
+safety button and can be attached as well if needed.
+More information on the pinout can be found in [this document](http://www.hex.aero/wp-content/uploads/2016/07/DRS_Pixhawk-2-17th-march-2016.pdf),
+page 17.
+
+![](../../assets/airframes/multicopter/Flamewheel_450/f450_setup_gps.JPG)
+
+| pin | Here+ GPS     | pin | Pixhawk 3 Pro GPS |
+| --- | ------------- | --- | ----------------- |
+| 1   | VCC_5V        | 1   | VCC               |
+| 2   | GPS_RX        | 2   | GPS_TX            |
+| 3   | GPS_TX        | 3   | GPS_RX            |
+| 4   | SCL           | 4   | SCL               |
+| 5   | SDA           | 5   | SDA               |
+| 6   | BUTTON        | -   | -                 |
+| 7   | BUTTON_LED    | -   | -                 |
+| 8   | GND           | 6   | GND               |
+
+## Parameters
+
+General documentation on how to setup your quad in *QGroundControl* can be found [here](https://docs.qgroundcontrol.com/en/).
+
+### Airframe
+
+Select the airframe **DJI Flame Wheel 450** in **QGC > Airframe > Quadrotor x**.
+
+![](../../assets/airframes/multicopter/Flamewheel_450/f450_setup_airframe.png)
+
+### Lidar-Lite
+
+To enable the *Lidar-Lite V3* (connected via I2C), the parameter `SENS_EN_LL40LS` has to be set to `2`. 
+
+> **Note** This can be done using the NuttX shell and the command `param set SENS_EN_LL40LS 2`. 
```

Comment:
@ChristophTobler @MaEtUgR I've just migrated the Flamewheel 450+RTK doc to the user guide. Is it still necessary to set these parameters via NuttX or can they now be set through QGC?

Can you also please have a quick scan over this and confirm it is still accurate?

Reply:
@hamishwillee Thanks! This is now possible with QGC. You have to choose `I2C`

Reply:
Thanks @ChristophTobler. I'll update and merge this now.